pkg/handlers/snapshots.go

```diff
@@ -65,6 +65,8 @@ type UpdateGlobalSnapshotSettingsRequest struct {
 	Other      *kotssnapshottypes.StoreOther  `json:"other"`
 	Internal   bool                           `json:"internal"`
 	FileSystem *FileSystemOptions             `json:"fileSystem"`
+
+	CACertData []byte `json:"ca_cert_data"`
```

Comment:
let's keep the naming convenction
```suggestion
	CACertData []byte `json:"caCertData"`
```